t/Util.pm

```diff
@@ -64,6 +65,45 @@ sub exec_unittest {
     die "failed to exec $fn:$!";
 }
 
+sub exec_mruby_unittest {
+    my $test_dir = path('t/00unit.mruby');
+    my $bin = path(bindir(), 'mruby-test/host/bin/mruby');
+    plan skip_all => "unit test:mruby binary $bin does not exist. Please build mruby for testing by `make mruby-test in your build directory`"
+        if ! -e $bin;
```

Comment:
Considering the fact that building H2O without mruby support is a legitimate option, It would be great if you could emit a more polite message when the tests need to be skipped.

Maybe you should run the code below (which can be found in other tests), and then check the existence of `mruby` binary (in which case, you should emit a hard error considering the fact that the server is built _with_ mruby support).
```
plan skip_all => 'mruby support is off'
    unless server_features()->{mruby};
```